content/terraform-docs-common/docs/plugin/terraform-plugin-protocol.mdx

```diff
@@ -28,6 +32,8 @@ Implementations include:
 * [tf5to6server](/terraform/plugin/mux/translating-protocol-version-5-to-6): A package to translate protocol version 5 providers into protocol version 6.
 * [tf6muxserver](/terraform/plugin/mux/combining-protocol-version-6-providers): A package to combine multiple protocol version 6 providers.
 
+The protocol v6 definition can be found [here](https://github.com/hashicorp/terraform/blob/main/docs/plugin-protocol/tfplugin6.proto).
```

Comment:
```suggestion
Refer to [Terraform Plugin RPC protocol version 6.10](https://github.com/hashicorp/terraform/blob/main/docs/plugin-protocol/tfplugin6.proto) in the Terraform repository for the protocol v6 definition.
```
Refer to the [style guide](https://github.com/hashicorp/web-unified-docs/blob/main/docs/style-guide/general/links.md#use-descriptive-link-text-that-explicitly-tells-readers-about-the-destination-content)